compiler-core/src/build/package_compiler.rs

```diff
@@ -594,28 +603,28 @@ pub(crate) fn module_name(package_path: &Utf8Path, full_module_path: &Utf8Path)
 #[derive(Debug)]
 pub(crate) enum Input {
     New(UncompiledModule),
-    Cached(CachedModule),
+    Cached((CachedModule, CacheMetadata)),
 }
 
 impl Input {
     pub fn name(&self) -> &EcoString {
         match self {
             Input::New(m) => &m.name,
-            Input::Cached(m) => &m.name,
+            Input::Cached(m) => &m.0.name,
         }
     }
 
     pub fn source_path(&self) -> &Utf8Path {
         match self {
             Input::New(m) => &m.path,
-            Input::Cached(m) => &m.source_path,
+            Input::Cached(m) => &m.0.source_path,
```

Comment:
same thing here

Reply:
Thanks! Fixed in commit 4789222ac8db5fe157576681547f5458f1508e76